test/k8sT/Policies.go

```diff
@@ -884,29 +883,45 @@ var _ = Describe("K8sPolicyTest", func() {
 				monitorFile := fmt.Sprintf(monitorFileName, uuid.NewUUID().String())
 
 				By("Starting monitor and generating traffic which should%s redirect to proxy", not)
-				monitorStop := kubectl.MonitorStart(helpers.CiliumNamespace, ciliumPod, monitorFile)
+				monitorRes, monitorCancel := kubectl.MonitorStart(helpers.CiliumNamespace, ciliumPod)
+
+				var monitorOutput []byte
+				body := func() bool {
+					monitorOutput = append(monitorOutput, monitorRes.CombineOutput().Bytes()...)
+
+					// Let the monitor get started since it is started in the background.
+					time.Sleep(2 * time.Second)
```

Comment:
We can drop the `sleep`s since we are using a timeout anyway (we can return `false` if exec is not successful)

Reply:
I switched this to not use WithTimeout.